### ISSUE TITLE
VOC2007 mAP Results

### ISSUE THREAD

**Issue by fatihbaltaci** [ppengtang/pcl.pytorch#27]
Hi, thank's for sharing your code. 

From the paper, mAP of PASCAL VOC 2007 test set is 45.3% with single model. I trained the model and got 43% with the shared hyper-parameters. 

Is there anything I should do to get the actual result?

Thanks.

**Comment by ppengtang** [comment]
Hi, using this repo, you should get 49.2% mAP on the PASCAL VOC 2007 test set. See #9 for some questions related to reproducing our results. You can also try the original Caffe implementation and combine with these two improvements (see #11 ) to get similar mAP.
